sgn.json

```diff
@@ -58,5 +58,44 @@
             "href": "http://mathworld.wolfram.com/Sign.html",
             "title": "Sign explained by Wolfram MathWorld"
         }
-    ]
+    ],
+    "process_graph": {
+        "gt0": {
```

Comment:
`process_id` and `arguments` are missing here? for `lt0` too?

Reply:
Whoops, fixed.